contributing.md

```diff
@@ -128,15 +128,19 @@ all of the tests still pass. To do so:
 1.  Build a distribution archive with `bazel build //:bazel-distfile`.
 
 2.  Unzip the created binary in a new empty directory and in that directory run
-    `bash compile.sh all`. This script does two things:
+    `bash compile.sh all`. This script does the following:
 
-    *   Rebuilds Bazel with `./compile.sh`, Bazel with the
-        `compile.sh` Bazel, and Bazel with the Bazel-built binary.
+    *   Rebuilds Bazel with `compile.sh` and with the Bazel-built binary.
```

Comment:
here we're creating 2 builds of Bazel. Is one of these Bazel with your changes and the other Bazel at head? Or are they both Bazel with the new changes but built in different ways?

Reply:
To be clear: the test suite is complete, to test bazel, there is no need to build bazel neither with bazel nor with compile.sh, `bazel test //src/... //third_party/ijar/... //scripts/...` should test everything already.